src/pevm.rs

```diff
@@ -44,15 +42,19 @@ pub enum PevmError {
     UnreachableError,
 }
 
+/// Represents the state transitions of the EVM accounts after execution.
+/// If the value is `None`, it indicates that the account is marked as self-destructed.
```

Comment:
`as self-destructed` -> `for removal` since we now cover multiple removal logic (self-destructed and EIP-161).